MobileWhipWhepClient.podspec

```diff
@@ -1,18 +1,18 @@
 Pod::Spec.new do |s|
-  s.name             = 'MobileWhepClient'
+  s.name             = 'MobileWhipWhepClient'
```

Comment:
I think that after this modification the React Native example will stop working, especially [this](https://github.com/software-mansion/react-native-whip-whep/blob/bcefd03c1bb1fa4202a28f71222f799690a706ec/examples/react-native/WhipWhepDemo/withLocalPathsForNativePackages.js#L81) linking